content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md

```diff
@@ -0,0 +1,57 @@
+---
+content_type: concept
+title: Perangkat untuk Memantau Sumber Daya
+---
+
+<!-- overview -->
+
+Untuk penyekalaan aplikasi dan memberikan Service yang handal, kamu perlu
+memahami bagaimana aplikasi berperilaku ketika aplikasi tersebut digelar. Kamu bisa memeriksa
+kinerja aplikasi dalam klaster Kubernetes dengan memeriksa Container,
+[Pod](/id/docs/user-guide/pods), [Service](/id/docs/user-guide/services), dan
+karakteristik klaster secara keseluruhan. Kubernetes memberikan detail
+informasi tentang penggunaan sumber daya dari aplikasi pada setiap level ini.
+Informasi ini memungkinkan kamu untuk mengevaluasi kinerja aplikasi kamu dan
+mengevaluasi di mana kemacetan dapat dihilangkan untuk meningkatkan kinerja secara keseluruhan.
+
+
+
+<!-- body -->
+
+Di Kubernetes, pemantauan aplikasi tidak bergantung pada satu solusi pemantauan saja. Pada klaster baru, kamu bisa menggunakan _pipeline_ [metrik sumber daya](#resource-metrics-pipeline) atau _pipeline_ [metrik penuh](#full-metrics-pipeline) untuk mengumpulkan statistik pemantauan.
+
+## _Pipeline_ Metrik Sumber Daya
+
+_Pipeline_ metrik sumber daya menyediakan sekumpulan metrik terbatas yang terkait dengan
+komponen-komponen klaster seperti _controller_ [Penyekala Pod Horisontal](/id/docs/tasks/run-application/horizontal-pod-autoscale), begitu juga dengan utilitas `kubectl top`.
```

Comment:
HPA is kubernetes object
and typo on link
```suggestion
komponen-komponen klaster seperti _controller_ [HorizontalPodAutoscaler](/id/docs/tasks/run-application/horizontal-pod-autoscaler), begitu juga dengan utilitas `kubectl top`.
```

Reply:
Ok!